frontend/src/actions/ocpActions.js

```diff
@@ -209,13 +208,19 @@ export const setOCPOtherSummaryFilter = () => (dispatch, getState) => {
   dispatch(tableReCalcValues());
 };
 
-export const getOCPSummary = () => (dispatch, getState) => {
-  const results = [...getState().ocp.filteredResults];
-
-  const countObj = calculateMetrics(results);
+export const getOCPSummary = (countObj) => (dispatch) => {
+  const other =
+    countObj["total"] -
+    ((countObj["success"] || 0) + (countObj["failure"] || 0));
+  const summary = {
+    othersCount: other,
+    successCount: countObj["success"] || 0,
+    failureCount: countObj["failure"] || 0,
```

Comment:
I wonder about the wisdom of assuming `0` when either of these fields is missing, although I suspect that doesn't happen routinely. Would it make more sense to just use `countObj?.success` and let it be `undefined`, or would that cause worse issues elsewhere?

Reply:
Then, in the `Metrics` tab under `undefined` will be displayed instead of `0`. 

Yeah, I could change the format the values in Metrics tab component. But, is it necessary ?

Reply:
There may not be a perfect solution. But if the `success` and `failure` fields *should* always be there, maybe even something like `countObj?.failure || "Error"`, instead of suggesting there are no failures? I'm a little less worried about the success case, actually; if the `success` field is missing, there probably *are* no successes. 😆

Reply:
Although I really don't like showing "0 failures" when the lack of the `failure` key suggests the test may have gone completely bad, I suppose we can worry about presentation aspects like that separately, and I'll close the conversation. Maybe nobody will care. 😆

Reply:
ES instance returns `{success:10}` when there are no failures so it's better to check for missing fields and setting it to `0` which would be easier to render the Metrics Tab

Reply:
Yeah, OK, that makes it less clear-cut. Fine -- I'll just mark this resolved.